src/App.tsx

```diff
@@ -15,47 +16,128 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+export function getReorderedGoods(
+  goods: string[],
+  sortType: SortType,
+  isReversed: boolean,
+) {
+  const visibleGoods = [...goods];
+
+  switch (sortType) {
+    case ('Alphabet'):
+      visibleGoods.sort((good1, good2) => (
+        good1.localeCompare(good2)));
+      break;
+    case ('Length'):
+      visibleGoods.sort((good1, good2) => (
+        good1.length - good2.length));
+      break;
+
+    default:
+      break;
+  }
```

Comment:
- use enum value here
- follow DRY , use sort just once
- default case should throw error
- return immediately from case